### PR TITLE
Add `monitor-`[`up`/`down`] icons

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Review by jguddas — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]
Nice work!

File: icons/monitor-up.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "tv",
+    "screen",
+    "display",
+    "upload",
+    "connect",
+    "remote"
```
Comment (karsa-mistmere) [inline icons/monitor-up.json:9]:
```suggestion
    "remote",
    "screen share"
```

File: icons/monitor-down.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "tv",
+    "screen",
+    "display",
+    "desktop",
+    "download",
+    "clone",
+    "version control",
+    "git"
```
Comment (karsa-mistmere) [inline icons/monitor-down.json:11]: I like the icon in general, but I'm not sure about these tags, I don't think I've ever seen this type of icon being used for git clone. 🤔

Comment (danielbayley) [reply icons/monitor-down.json:11]: Hmm, I was going off this…

Untitled

Comment (karsa-mistmere) [reply icons/monitor-down.json]: I think that's maybe more about GitHub _Desktop_ and less about `git clone`. 😄 

Same with the other "clone" icon in this dropdown, which stands in for "git clone" _using CLI_:

image

**Pull request merged**